test/models/club_test.exs

```diff
@@ -2,17 +2,42 @@ defmodule Gatherto.ClubTest do
   use Gatherto.ModelCase
 
   alias Gatherto.Club
+  alias Gatherto.Athlete
 
-  @valid_attrs %{description: "some content", title: "some content"}
+  @valid_attrs %{description: "some content",
+                 title: "some content",
+                 location: "Cleveland, OH"}
   @invalid_attrs %{}
 
-  test "changeset with valid attributes" do
-    changeset = Club.changeset(%Club{}, @valid_attrs)
-    assert changeset.valid?
-  end
+  describe "a changeset" do
```

Comment:
I'm not sure if this is the best describe. Although accurate, I realized halfway through writing this that you can't nest Describe.